pocs/tests/test_mount.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from pocs.mount import create_mount_from_config, AbstractMount
+from pocs.utils.error import MountNotFound
+from pocs.utils.location import create_location_from_config
+
+
+def test_bad_mount_port(config):
+    conf = config.copy()
+    conf['mount']['serial']['port'] = '/dev/'
+    with pytest.raises(MountNotFound):
+        create_mount_from_config(conf)
+
+
+@pytest.mark.without_mount
+def test_bad_mount_driver(config):
+    conf = config.copy()
+    conf['mount']['driver'] = 'foobar'
+    with pytest.raises(MountNotFound):
+        create_mount_from_config(conf)
+    conf['mount']['driver'] = 1234
+    with pytest.raises(MountNotFound):
+        create_mount_from_config(conf)
+
+
+def test_create_mount_with_earth_location(config):
```

Comment:
Let's also add a basic `test_create_mount_from_config` that doesn't use the `earth_location`.